control/pid_longitudinal_controller/src/pid_longitudinal_controller.cpp

```diff
@@ -163,7 +163,6 @@ PidLongitudinalController::PidLongitudinalController(rclcpp::Node & node)
   m_min_jerk = node.declare_parameter<double>("min_jerk");  // [m/s^3]
 
   // parameters for slope compensation
-  m_use_traj_for_pitch = node.declare_parameter<bool>("use_trajectory_for_pitch_calculation");
```

Comment:
Because slope compensation won't be done with respect to the user's option. While driving the ego, it is always needed that the slope should be calculated from future position projection to the trajectory of the vehicle.

The pitch information from the sensing only is usable when the vehicle is in the stopped state.